modin/pandas/base.py

```diff
@@ -1473,17 +1473,30 @@ def kurt(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
         Returns:
             kurtosis : Series or DataFrame (if level specified)
         """
+        axis = self._get_axis_number(axis)
         if level is not None:
-            return self._default_to_pandas(
-                "kurt",
-                axis=axis,
-                skipna=skipna,
-                level=level,
-                numeric_only=numeric_only,
-                **kwargs,
+            from .series import Series
+
+            func_kwargs = {
+                "axis": 0,
+                "skipna": skipna,
+                "level": level,
+                "numeric_only": numeric_only,
+            }
+            if isinstance(self, Series):
+                return (
+                    self.groupby(level=level, axis=axis, sort=False)
+                    .apply(func="kurt", **func_kwargs)
+                    .rename(None)
+                )
+
+            new_modin_frame = self._query_compiler._modin_frame._apply_full_axis(
```

Comment:
API layer should only rely on `_query_compiler` interface. Omnisci will not have a `_modin_frame`.

We can add a `kurt` to the query compiler interface to resolve this.

Reply:
Now `_query_compiler.apply` function is called. Is it appropriate solution?